core/internal/cltest/cltest.go

```diff
@@ -1683,3 +1687,49 @@ func ClearDBTables(t *testing.T, db *sqlx.DB, tables ...string) {
 	err = tx.Commit()
 	require.NoError(t, err)
 }
+
+// FlagSetApplyFromAction applies the flags from action to the flagSet.
+// `parentCommand` will filter the app commands and only applies the flags if the command/subcommand has a parent with that name, if left empty no filtering is done
+func FlagSetApplyFromAction(action interface{}, flagSet *flag.FlagSet, parentCommand string) {
+	cliApp := cmd.Client{}
+	app := cmd.NewApp(&cliApp)
+
+	foundName := parentCommand == ""
+	actionFuncName := getFunctionName(action)
```

Comment:
I think it would help to align the other pieces too:
```suggestion
	actionFuncName := getFuncName(actionFunc)
```

Reply:
Fixed in [5e861ee](https://github.com/smartcontractkit/chainlink/pull/8173/commits/5e861eead31f095b03094245c6692f9755ad0f15)